tests/test_init.py

```diff
@@ -173,28 +170,6 @@ def on_service_state_change(zeroconf, service_type, state_change, name):
         # close our zeroconf which will close the sockets
         zc.close()
 
-        # pop the big chunk off the end of the data and send on a closed socket
```

Comment:
This test checks to see if we write to a closed socket.  That can't happen with asyncio since we aren't writing directly in most cases